falcon/media/json.py

```diff
@@ -12,19 +12,61 @@ class JSONHandler(BaseHandler):
 
     This handler uses Python's :py:mod:`json` by default, but will
     use :py:mod:`ujson` if available.
+
+    Keyword Arguments:
+        dumps (func): the function to use when serializing JSON responses
+
+        loads (func): the function to use when deserializing JSON requests
+
+        dumps_kwargs (dict): keyword arguments passed to ``dumps`` when serializing to JSON.
+            ``ensure_ascii`` is set to ``false`` by default. To override this setting, pass ``True``
+            explicitly.
+
+        loads_kwargs (func): keyword arguments to pass to ``loads`` when deserializing from JSON.
+
+
+    You can override the JSON library used by changing the ``dumps`` and ``loads`` functions::
+
+        import falcon
+        from falcon import media
+        import rapidjson
+
+
+        json_handler = media.JSONHandler(
+            dumps=rapidjson.dumps,
+            loads=rapidjson.loads,
+            dumps_kwargs=dict(ensure_ascii=False)
+        )
+        extra_handlers = {
+            'application/json': json_handler,
+            'application/json; charset=UTF-8': json_handler,
```

Comment:
Probably outside of scope of this issue, but do we need to single out this ``application/json; charset=UTF-8``? 
And I think I've seen an issue or a code comment somewhere to phase this explicit charset out, wouldn't ``application/json`` capture the latter as well?